charts/kubescape-cloud-operator/templates/node-agent/daemonset.yaml

```diff
@@ -3,28 +3,28 @@
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: {{ .Values.nodeAgent.daemonset.name }}
+  name: {{ .Values.nodeAgent.name }}
```

Comment:
I did not see the related change in the values file

Reply:
https://github.com/kubescape/helm-charts/pull/196/files#diff-233118cd3763c2ae09c5170bc1382aff6de42604596e47881db3b11672f21a31R565